here/here_aggregation.md

```diff
@@ -0,0 +1,252 @@
+# Table of Contents  
+- [What is HERE data?](#what-is-here-data-?)  
+- [Input Parameters](#input-parameters)  
+    - [Geometry](#Geometry)
+    - [Time Range](#time-range)
+    - [Date Range](#date-range)
+- [Output Parameters](#output-parameters)  
+- [Aggregation](#aggregation)
+    - [Using congestion tables](#using-congetion-tables)
+    - [Using raw speed table](#using-raw-speed-table)
+
+## What is HERE data?
```

Comment:
What is HERE data should be merged into the `here/readme.md` and this aggregation readme should be referenced from it.